ferris_elf/__init__.py

```diff
@@ -559,7 +510,7 @@ async def handle_dm_commands(client: "MyBot", msg: discord.Message) -> None:
             return
 
         for file in onlyfiles:
-            with open(join(day_path, file), "r") as f:
+            with open(join(day_path, file), "rb") as f:
```

Comment:
Why the `rb`?

Reply:
discord.py File typings dont accept a TextIOStream so i changed it to BinaryIO